specification/web/resource-manager/Microsoft.Web/stable/2016-06-01/examples/customApisList.json

```diff
@@ -0,0 +1,37 @@
+{
+  "parameters": {
+    "subscriptionId": "34adfa4f-cedf-4dc0-ba29-b6d1a69ab345",
+    "resourceGroupName": "testResourceGroup",
+    "api-version": "2016-06-01"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "value": [
+          {
```

Comment:
Could you add a second entry to the value array, that second entry being an example of a Soap api? I don't fully recall if Soap passthrough will further be its own apiType - if so do have an example of each in the list response.

Reply:
Wouldn't the response object be the same? Unless the soap version exposes different fields in the response, I'm not sure it adds anything.

Reply:
there will at least be difference that SOAP custom API will have a (non-null) WSDL which neither REST nor pass-through will have.

Reply:
Added a Soap one, with the WSDL section filled in